ptime.go

```diff
@@ -369,6 +369,40 @@ func (t *Time) SetUnix(sec, nsec int64, loc *time.Location) {
 	t.SetTime(time.Unix(sec, nsec).In(loc))
 }
 
+// norm returns nhi, nlo such that
+//	hi * base + lo == nhi * base + nlo
+//	0 <= nlo < base
+func norm(hi, lo, base int) (nhi, nlo int) {
```

Comment:
don't use name return types

Reply:
its copied from time package exactly as it is